compile.sh

```diff
@@ -47,14 +46,18 @@ Executables:
   ${YELLOW}--artss_gpu_benchmark${NC}               \t Executable artss_gpu_benchmark
 
 Other:
-   ${YELLOW}-c${NC}
```

Comment:
the blank is on purpose to align the text

Reply:
done in 13f32dd

Reply:
if you change it from blank ${YELLOW}-c to ${YELLOW} blank -c, you should do it thoroughly. or change it back.